pulumi/github/teams/.gitignore

```diff
@@ -1,4 +1,7 @@
-
 *.pyc
 venv/
 __pycache__/
+
+# sensitive data
+Pulumi*yaml
+*.txt
```

Comment:
are we sure we don't want to track the `requirements.txt`

Reply:
The `requirements.txt` file was already part of this PR so it wouldn't get ignored in the future, but I added an explicit exception in the gitignore file.